### ISSUE TITLE
outlook-set-signature - Unable to get to run

### ISSUE THREAD

**Issue by nmbgeek** [OfficeDev/Office-Add-in-samples#575]
Running into a few issues getting started.  I initially copied outlook-set-signature and started modifying but then began running into the below errors once I was ready to start testing out my changes.  To make sure it was nothing that I had directly caused I then created a new npm project with an unmodified copy of outlook-set-signatures and am still running into the same issues.

**Question 1**
When trying to run a fresh unmodified copy of the outlook-set-signature sample and using:
```
npm install
npm run start
```

The add-in is loaded in Outlook however when creating a new email and clicking on the add-in an error is displayed along with `Cannot GET /src/taskpane/HTML/editsignature.html`
The webpack console appears to not have any errors and is running along with the server on port 3000 (localhost:3000 returns `Cannot GET /`
Webpack console displays this:

```
You already have trusted access to https://localhost.
Certificate: C:\Users\xxx\.office-addin-dev-certs\localhost.crt
Key: C:\Users\xxx\.office-addin-dev-certs\localhost.key
<i> [webpack-dev-server] Project is running at:
<i> [webpack-dev-server] Loopback: https://localhost:3000/
<i> [webpack-dev-server] Content not from webpack is served from 'C:\Users\xxx\Nextcloud\coding\microsoft\outlook-add-in-set-signature\public' directory
<i> [webpack-dev-middleware] wait until bundle finished: /src/taskpane/HTML/editsignature.html?_host_Info=Outlook$Win32$16.02$en-US$$$$2
assets by path *.js 1.8 MiB
  assets by info 21.2 KiB [immutable] 7 assets
  + 5 assets
assets by path assets/ 125 KiB
  asset assets/outlook-set-signature-overview.png 121 KiB [emitted] [from: assets/outlook-set-signature-overview.png] [copied]
  asset assets/sample.json 2.3 KiB [emitted] [from: assets/sample.json] [copied]
  + 2 assets
assets by path *.html 5.78 KiB
  asset assignsignature.html 3.46 KiB [emitted]
  asset editsignature.html 1.52 KiB [emitted]
  asset autorunweb.html 822 bytes [emitted]
assets by path *.css 9.25 KiB
  asset 7f5eb175e32e87723ff6.css 7.64 KiB [emitted] [immutable] [from: src/taskpane/CSS/assignsignature.css] (auxiliary name: assignpane)
  asset 91ab987be678ba9134bf.css 1.61 KiB [emitted] [immutable] [from: src/taskpane/CSS/editsignature.css] (auxiliary name: editpane)
asset manifest.json 10.3 KiB [emitted] [from: manifest.json] [copied]
runtime modules 110 KiB 48 modules
modules by path ./node_modules/core-js/ 541 KiB 520 modules
modules by path ./src/ 20.4 KiB (javascript) 30.4 KiB (asset) 14 modules
modules by path ./node_modules/webpack-dev-server/client/ 56.8 KiB 12 modules
modules by path ./node_modules/webpack/hot/*.js 4.59 KiB
  ./node_modules/webpack/hot/dev-server.js 1.88 KiB [built] [code generated]
  + 3 modules
modules by path ./node_modules/html-entities/lib/*.js 81.3 KiB
  ./node_modules/html-entities/lib/index.js 7.74 KiB [built] [code generated]
  ./node_modules/html-entities/lib/named-references.js 72.7 KiB [built] [code generated]
  + 2 modules
./node_modules/regenerator-runtime/runtime.js 24.6 KiB [built] [code generated]
./node_modules/html-loader/dist/runtime/getUrl.js 530 bytes [built] [code generated]
./node_modules/ansi-html-community/index.js 4.16 KiB [built] [code generated]
./node_modules/events/events.js 14.5 KiB [built] [code generated]
webpack 5.77.0 compiled successfully in 23213 ms
```

**Question 2**
Next when running `npm run start:unified` I am sent to my Microsoft sign-in and after signing in the callback appears to go to localhost:53XXX which shows the error `localhost sent an invalid response` along with `ERR_SSL_PROTOCOL_ERROR` and the npm cli  shows a timeout error.

**Comment by davidchesnut** [comment]
Hi @nmbgeek, thanks for letting us know about these issues. I will take a look at the sample and see if I can repro this behavior. 

Thanks!
David

**Comment by davidchesnut** [comment]
Hi @nmbgeek, the localhost manifest had some incorrect URI locations. I fixed them in PR #580. Using these changes you should be able to get it working.

Thanks!
David
